src/main/kotlin/bsu/cc/views/MainView.kt

```diff
@@ -50,15 +50,15 @@ class MainView : View("Conflict Checker") {
                 }
                 button("Print Test") {
                     setOnAction{
-                        ParserDemo.demoParse(fileNameField.text)
+                        DemoParser.demoParse(fileNameField.text)
                     }
                 }
             }
         }
         bottom {
             hbox {
                 label("File Name")
-                fileNameField = textfield()
+                fileNameField = textfield("src\\main\\resources\\Spring 2019 Validation Report Example.xlsx")
```

Comment:
Not sure if you were aware, but kotlin supports raw strings. though the """ would have taken more keystrokes in this cases haha.